src/components/Router.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import Home from './Home';
+import Navbar from './Navbar';
+import Calculator from './Calculator';
+import Quotes from './Quotes';
+import ErrorPage from './Errorpage';
+
+function RouterNav() {
+  return (
+    <>
+      <BrowserRouter className="app-container">
+        <Navbar />
+        <Routes>
+          <Route exact path="/" element={<Home />} />
+          <Route exact path="calculator" element={<Calculator />} />
```

Comment:
- [ ] Kindly 🙏🏻 adhere to the project requirement, according to which you should follow the bare minimum requirement to make your design as per the wireframe provided by `Microverse` for this project. Your Calculator should appear on the right-hand side of the screen and your `Let's do some math` text is missing, given in the wireframe. The rest of everything is up to you like, the colour etc. Please have a look at the below comparison to understand the things.

### Design Comparison
<table>
<tr>
<th>Provided Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

<img width="1066" alt="math_magicians_calculator" src="https://github.com/alex1779/math-magicians/assets/66476329/b4b9573d-c47b-42eb-be28-42d04418dafd">


</td>
<td>

![Screenshot from 2023-09-27 03-31-52](https://github.com/alex1779/math-magicians/assets/66476329/eef4b1ec-60dc-4172-9056-acea06379826)


</td>
</tr>
</table>